docs/CI & Tests/Mint.md

```
@@ -0,0 +1,56 @@
+## Mint GitHub actions
+
+NooBaa CI contains 2 Github actions that build, configure and run Mint. These Github actions run automatically on every PR and on every push, and can run by workflow dispatch manually.
+* [Mint Tests](../../.github/workflows/mint-tests.yaml) - Based on NooBaa Tester image, runs Mint on standard NooBaa (db configuration).
+* [Mint NC Tests](../../.github/workflows/mint-nc-tests.yaml) - Based on NooBaa Tester image, runs Mint on non-containerized NooBaa (ConfigFS configuration).
+
+Our next goal is to add longer Mint runs as part of NooBaa's nightly CI process.
+
+## Mint Makefile Targets
+
+One can run Mint tests on NooBaa using Mint Makefile targets - 
+* `make run-mint` - Based on NooBaa Tester image, runs Mint on standard NooBaa (db configuration).
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)